modules/distr-tracing-tempo-getting-the-tempostack-instance-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-tempo-stack-logs_{context}"]
+= Getting the TempoStack instance logs
+
+You can get the logs from a TempoStack component as follows.
```

Comment:
Sort of a side note / something for consideration maybe from a reader's perspective...

I've noticed that there isn't a huge amount of conceptual information about what a TempoStack _is_ exactly in these docs. I would assume from this context that it's referring to something that is a Kubernetes object, but if that's the case, it should be formatted throughout the docs as "`TempoStack` object", or "`TempoStack` CR" etc per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references

In these docs, it seems to be sort of inconsistently referred to in a couple of different ways:
-  TempoStack component
- TempoStack instance

As a reader, I think this is unclear / confusing, because it's not really clear if what's being referred to is an instance of a product, e.g. like an OCP cluster, or a specific Kubernetes object, like a `TempoStack` CR.

Maybe the docs can be reworked a bit to make it clearer what's being referred to here, or refer to it more consistently, and maybe even add a link to some conceptual docs as an additional resource?

Reply:
Thank you, Ashleigh, I will think more about this for both Tempo and OTEL. I've added a link to this comment to one of my to-do Jiras in this area of docs improvement.